src/Traits/LanguageHydrationTrait.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
Just up for discussion, not necessary a change request; but can we just use this in the abstract request ? 

I would imagine that we could make an "AbstractLanguageRequest" or just simply extend the AbstractRequest to include, as a good amount of the requests should likely be sending the lang flag long term; probably? Just food for thought, can discuss in pair.

Reply:
Thinking is, we won't have to remember to pull in a trait on something we do more often then naught, but looking for feedback.